services/ui-src/src/components/pages/GetStarted/McparGetStartedPage.tsx

```diff
@@ -129,4 +185,20 @@ const sx = {
       marginRight: "-0.25rem",
     },
   },
+  smallSectionHeading: {
+    marginBottom: "1rem",
+  },
+  tableHeading: {
+    Th: {
+      fontSize: "var(--chakra-fontSizes-sm)",
+      "text-transform": "none",
+      "padding-left": "0px",
+      color: "var(--chakra-colors-gray-500)",
+    },
+  },
+  tableBody: {
+    Td: {
+      "padding-left": "0px",
+    },
+  },
```

Comment:
Question: Is there a reason these have to be `"text-transform"` instead of `textTransform`? Forgive my ignorance if so.

Also, instead of using `var(--...)` we should be able to just use `fontSize: "sm"` and `color: palette.gray` to access variables exported via Chakra's theme in theme.ts.

Reply:
No reason, just preference. Can change. 

I'm fairly new to Chakra, awesome that I can access the palette that way!